### PR TITLE
ver 1.0

### PR DESCRIPTION
**Pull request by suhanovgleb**

### CODE REVIEW

File: ReflectionTest/ReflectionConsoleTest/Program.cs [hunk 1]
```diff
@@ -12,62 +12,61 @@ class Program
...
 
-                    if (typelist.Any())
-                    {
-                        break;
-                    }
-                }
+        public static bool GetReflectionInfo(string assemblyNamespace)
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            IEnumerable<Type> types = new Type[0];
```
Comment (DotNetPart) [inline ReflectionTest/ReflectionConsoleTest/Program.cs]: new Type[0]; is created and trown away after variable is assigned is Linq enumerable.

File: ReflectionTest/ReflectionConsoleTest/Program.cs [hunk 2]
```diff
@@ -12,62 +12,61 @@ class Program
...
-                }
+            users.AddUser(user);
+            users.DeleteUser(users.FindByLogin("tsar"));
+            users.DeleteUser(user);
 
-                users.AddUser(user);
-                users.DeleteUser(users.FindByLogin("tsar"));
-                users.DeleteUser(user);
-                users.DeleteUser(user);
+            GetReflectionInfo("SomeLibrary");
```
Comment (DotNetPart) [inline ReflectionTest/ReflectionConsoleTest/Program.cs]: Normally assemblies have a lot of different namespaces, but it's ok for now.

File: ReflectionTest/ReflectionConsoleTest/Program.cs [hunk 3]
```diff
@@ -12,62 +12,61 @@ class Program
...
                 {
-                    throw new Exception("Сan't find the target assembly.");
+                    break;
                 }
+            }
+
+            if (!types.Any())
+            {
+                return false;
+            }
```
Comment (DotNetPart) [inline ReflectionTest/ReflectionConsoleTest/Program.cs]: No need to break or return false. If Where returns empty enumerable you'll just never get into foreach.

Comment (suhanovgleb) [reply ReflectionTest/ReflectionConsoleTest/Program.cs]: Okay, I'll remember that. But break was used to ensure that the value types were not assigned null at further passes.

File: ReflectionTest/ReflectionConsoleTest/Program.cs (hunk 2 above)
Comment (suhanovgleb) [reply ReflectionTest/ReflectionConsoleTest/Program.cs]: Ok.

File: ReflectionTest/ReflectionConsoleTest/Program.cs (hunk 1 above)
Comment (suhanovgleb) [reply ReflectionTest/ReflectionConsoleTest/Program.cs]: I need to initialize "types", so I assigned a null.

**Pull request closed without merging**